### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by vr8hub** [chapkachapka/lady-gregory_w-b-yeats_the-unicorn-from-the-stars#1]
(As you well know, you have more experience with drama productions than I do :), so forgive the possibly ignorant questions. Since IA is down, I found what I believe is the same edition you used on [Google](https://www.google.com/books/edition/Plays_in_Prose_and_Verse/vpBEAQAAMAAJ).

- [x] Typogrify makes a change to the long description that should be accepted.
- [x] In Act 1, Father John has an "right-bracket" direction (_Goes in and shuts door._) in the scan that is a standalone direction in the text. Same a bit later (_Goes Out._). In Act 2, Johnny has a (_Points at_ Martin) right-bracket direction in the scan, but this time it's inline in the text. According to the "How to conquer…" document that Emily put together, right-bracket directions should be inline at the end of the dialog. That says to me that all three of the above should be inline rather than the first two being standalone. There are also a few others I saw in the same situation. Am I misunderstanding something?

**Comment by vr8hub** [comment]
Also, it would be helpful if you would either submit a PR to the manual to split the stage directions CSS out from the table CSS, or just keep the two together as the manual has it. As it is, your reviewers have to go back and forth to make sure everything is present and the same. Thanks!

**Comment by chapkachapka** [comment]
A lot of the drama formatting document is based specifically on Shakespeare and the Shakespeare editions that SE used. In those you usually have very basic stage directions--like "How to conquer" says, often literally just "exit" or "exeunt".

In this edition pretty much every stage direction is formatted with just an opening bracket, unless it's followed by more dialogue. This goes beyond just the "exit" type directions in the guide--for example, on page 278, Johnny's line is followed by "[Martin comes back with lighted lamp". I guess in my mind this is separate from what the Conquer doc is talking about, where it is talking about just "exit and exeunt" stage directions. Let me know if you think this is something we should clarify on the mailing list, and I'll post a few examples to get feedback.

**Comment by vr8hub** [comment]
Right, I saw that. But the reason I looked in the first place is because the two examples I gave and a few others look like they should be inline. (As with the third example.) So regardless of right bracket or not, why wouldn’t those be inline?

**Comment by chapkachapka** [comment]
I think the difference in the three examples you picked out was that the third one was left inline in the PG text I was working from. The others were broken out on their own lines.

In general, I see the "right bracket" notation as a way to save space on the page; it's a way to avoid putting the direction on the line. To me, the real division is between directions with open and close brackets, showing that they're inline, and ones with just open brackets, which are on their own line.

I think going through the semantics of each direction to decide whether it "feels" more like an inline or between-line direction isn't particularly helpful. I think the question is, does it need to be inline? Otherwise I think it's clearer and more consistent with modern publishing practices to split directions onto their own lines.

**Comment by chapkachapka** [comment]
Okay; per Emma I've redone the stage directions.

**Comment by vr8hub** [comment]
Great, looks good. One more question of ignorance: in Act 2, in the stage directions of the beggars pushing in a barrel, their cry is in quotes and normal font in the scans, i.e. emphasized vs the italicized directions. Is that normal, i.e. is dialog in stage directions always formatted opposite the surrounding directions, or was it done specifically here, as emphasis? I couldn't find another instance of dialog in directions in the play to check.

**Comment by chapkachapka** [comment]
Dialogue in stage directions is pretty unusual in general, and play formatting is not super standardised in general, but no, it definitely isn't always emphasised. I left the emphasis out under the general rule in SEMoS that using both emphasis and quotes is generally not necessary.

**Comment by vr8hub** [comment]
The general approach is correct, but it's for _italicized_ text, not necessarily emphasized. That is, most of the time that text is italicized and quoted, it's not for emphasis, which is why we only keep the quoted. There are plenty of instances when it _is_ emphasis (from the context), in which case we convert the italics to emphasis and keep both.
That's why I asked; if direction dialogue is not _normally_ emphasized, then it's possible that it's emphasized here on purpose, i.e. as actual emphasis, not just because it's quoted. And in that case, we would emphasize it as well.
So, if you think this dialog is actual emphasis, then it should have em tags. If not, the way it is is fine. I'll leave it to you to decide.

**Comment by chapkachapka** [comment]
In this case I would leave it as it; I don't see emphasis as making sense for the whole phrase ("Hi! for the noble master!").
